### PR TITLE
Release 2.1.1

### PR DESCRIPTION
**Pull request by jakirkham**
Bumps the version to the latest `conda-build` available `2.1.1`.

### CODE REVIEW

**Comment by jakirkham** [comment]
When you have a chance, could you please review this, @msarahan? Would like to make sure I have everything correct for the latest release.

**Comment by jakirkham** [comment]
> Conda build is not currently compatible with conda 4.3. might be worth pinning to avoid problems there until I can fix it. Otherwise LGTM.

SGTM. Would the solver still try to downgrade `conda-build` to escape that pinning? If so, we might want to add that pinning to `conda-forge-build-setup` too.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,17 +30,20 @@ requirements:
     - setuptools
 
   run:
+    - beautifulsoup4
+    - chardet
+    - conda  >=4.1,<4.3
```
Comment (jakirkham) [inline recipe/meta.yaml]: Pinned like so. Is this what you had in mind?

Comment (msarahan) [inline recipe/meta.yaml]: Should be fine for now. I'll keep you posted on when conda build gets fixed for this.

**Comment by msarahan** [comment]
Ready for merge when appveyor passes.

**Comment by jakirkham** [comment]
Thanks Mike. Looks like it checks out.

**Pull request merged**